launch.go

```diff
@@ -167,20 +163,11 @@ func redirectURL(w http.ResponseWriter, r *http.Request) {
 func quickLauncherHandler(w http.ResponseWriter, r *http.Request) {
 	hostURL := settings.Get("SURVEY_RUNNER_URL")
 	accountServiceURL := getAccountServiceURL(r)
-	AccountServiceLogOutURL := getAccountServiceURL(r)
 	urlValues := r.URL.Query()
 	schemaURL := urlValues.Get("schema_url")
-	version := urlValues.Get("version")
```

Comment:
I'm not sure if it's still in use, but there's [a bit in the readme](https://github.com/ONSdigital/eq-questionnaire-launcher#:~:text=The%20optional%20query%20parameter%20version%20can%20be%20added%20to%20the%20quick%20launch%20url%20which%20allows%20for%20the%20launch%20payload%20structure%20to%20be%20specified.) about quick-launch with an optional `version` in the query string. Would that need to be amended/removed too?

Reply:
We still want the ability to support being able to set other versions, even if we only support v2 for now, so I guess this can stay